etcdutl/etcdutl/migrate_command.go

```diff
@@ -95,7 +95,11 @@ func (o *migrateOptions) Config() (*migrateConfig, error) {
 	c.be = backend.NewDefaultBackend(GetLogger(), dbPath)
 
 	walPath := datadir.ToWALDir(o.dataDir)
-	w, err := wal.OpenForRead(c.lg, walPath, walpb.Snapshot{})
+	lastSnapshot, err := getLastSnapshotIndex(walPath)
```

Comment:
Can we propagate `c.lg` into `getSnapshotIndex` and eventually to `ValidSnapshotEntries`? It'll be good to log errors out that happen in those calls.